### DISCUSSION TITLE
Direction Date

### DISCUSSION THREAD

**Discussion by f4gqk** [poblabs/weewx-belchertown#786] [Q&A] [answered]
hello, 

I would like to change the direction of the date in the forecast part for the 24h part.

To transform into 03/06, 04/06, 06/06.....

![formatdate](https://user-images.githubusercontent.com/53508157/171800712-207c1677-1feb-45a3-9c47-a0c3d8abbe55.jpg)

Website is : http://www.meteo-guignes.fr

How to do ????

Thanks

Sebastien

**Comment by ThePeePs** [comment, marked as answer]
in either `skin.conf` or in your `weewx.conf` settings for the Belchertown skin, under Labels -> Generic change this to be the format that you'd like.

Default:
```
[Labels]
    [[Generic]]
        time_forecast_date = "ddd M/DD"
```
Your ask:
```
[Labels]
    [[Generic]]
        time_forecast_date = "ddd DD/MM"
```
d = day of week name (3 for short name, 4 for full name)
D = day of month (DD if you want a leading zero)
M = number of the month (MM if you want a leading zero, MMMM for the full name of the month)

All of the default settings can be found in [skin.conf](https://github.com/poblabs/weewx-belchertown/blob/master/skins/Belchertown/skin.conf#L601-L615) I suggest making the changes to your `weewx.conf` (add an extra square bracket to my example) file, as the `skin.conf` can change as new features are added, and code is updated.

Hope this helps.

**Comment by f4gqk** [comment]
Thanks.

It's OK
